prow/pod-utils/decorate/podspec.go

```diff
@@ -212,10 +213,14 @@ func decorate(spec *kube.PodSpec, pj *kube.ProwJob, rawEnv map[string]string) er
 		GcsCredentialsFile: fmt.Sprintf("%s/service-account.json", GcsCredentialsMountPath),
 		DryRun:             false,
 	}
-	initUploadConfigEnv, err := initupload.Encode(initupload.Options{
-		Log:     cloneLog,
+
+	initUploadOptions := initupload.Options{
 		Options: &gcsOptions,
-	})
+	}
+	if willCloneRefs {
+		initUploadOptions.Log = cloneLog
```

Comment:
nit: We could get rid of the `cloneLog` variable entirely now and just use the value `fmt.Sprintf("%s/clone.json", LogMountPath)`.

Reply:
Why? We still share the value between `clonerefs` and `initupload` and it is still wrong to have those two utilities looking at different files for the log

Reply:
Ah I forgot that the value was used above as well. This is fine then.